_layouts/doc_page.html

```diff
@@ -52,7 +52,12 @@
 
     <script>
     $(function() {
-      $(".toc").load("/docs/{{ version }}/toc.html");
+      $toc = $(".toc");
+      $toc.load("/docs/{{ version }}/toc.html", function () {
+        $toc.find('a').each(function(idx, el) {
+          el.href = el.href.replace('replace-with-version', '{{ version }}');
```

Comment:
btw, I thought that using JS here was kind of gross, but probably okay since we already use JS to load the toc in the first place. but a less gross approach would be cool.